dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/AuthoritiesControllerTest.java

```diff
@@ -46,9 +45,22 @@ class AuthoritiesControllerTest extends DhisControllerConvenienceTest {
   void testGetAuthorities() {
     JsonArray systemAuthorities = GET("/authorities").content().getArray("systemAuthorities");
     assertTrue(systemAuthorities.size() > 10);
-    // its sorted
-    JsonObject all = systemAuthorities.getObject(0);
-    assertEquals("ALL", all.getString("id").string());
-    assertEquals("ALL", all.getString("name").string());
+
+    assertTrue(
+        range(1, systemAuthorities.size())
+            .allMatch(
+                i ->
+                    systemAuthorities
+                            .getObject(i - 1)
+                            .getString("name")
+                            .string()
+                            .toLowerCase()
+                            .compareTo(
+                                systemAuthorities
+                                    .getObject(i)
+                                    .getString("name")
+                                    .string()
+                                    .toLowerCase())
+                        < 0));
```

Comment:
If I get this correct you want to make sure they are all different when lower-cased and maybe also sorted?
I think the better way to do this is to use `stringValues()` on the JsonArray, giving you a `List<String>`. 
You then can map `toLowerCase`. Then sort a copy and compare that for equals, also copy it in a set and compare that for size to make sure there are no duplicates.